src/main/java/com/example/foode/Offer/OfferRepository.java

```diff
@@ -0,0 +1,14 @@
+package com.example.foode.Offer;
+
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.Optional;
+
+public interface OfferRepository extends JpaRepository<Offer, Long> {
+
+    Page<Offer> findAllByUserId(Long userId, Pageable pageable);
+
+    Page<Offer> findAllByLocation(String location, Pageable pageable);
```

Comment:
i would add one extra method do get all offers -> its probably what we want to send on main offer page in first moment

Reply:
I thought that we should display offers from specific location on main page, that's why I added `findAllByLocation`. Default location could be Warsaw (or another city) and user would be able to change it